WcaOnRails/app/models/competition.rb

```diff
@@ -1561,6 +1572,15 @@ def self.search(query, params: {}, managed_by_user: nil)
       competitions = Competition.visible
     end
 
+    if params[:continent].present?
+      continent = Continent.find(params[:continent])
```

Comment:
Maybe `params.require(:continent)`?

Reply:
Hm is this necessary even after ensuring `params[:continent].present?`